metrics/resourcemanager.go

```diff
@@ -24,4 +24,12 @@ var (
 		Name:      "ema_cpu_usage",
 		Help:      "exponential moving average of CPU usage",
 	})
+	// PoolConcurrencyCounter means how much concurrency in the pool
+	PoolConcurrencyCounter = prometheus.NewGaugeVec(
```

Comment:
Will you modify the grafana dashboard in another PR?

Reply:
Yes, I will.